eng/deploy.yaml

```diff
@@ -204,10 +204,20 @@ stages:
       artifact: ScenarioTests
     - download: Build
       artifact: PackageArtifacts
+    - download: Build
+      artifact: Maestro.ScenarioTests
+
     - task: NuGetToolInstaller@1
       displayName: Use NuGet
       inputs:
         versionSpec: 5.3.x
+
+    - tast: UseDotNet@2
```

Comment:
What is tast? :) Please tell me you tested these changes to yaml files.

Reply:
Its a typo-ed task.